.github/workflows/build-workflow-matrix.json

```diff
@@ -5,6 +5,9 @@
   {
     "image-name":"api"
   },
+  {
+    "image-name": "api-dev"
+  }
```

Comment:
I wonder whether we want to build that image on every release, or just when building the merge builds, something like we are doing in `- name: Pull cache, build and push test image` step in build.yaml